### PR TITLE
Feat(browser): consolidate browser checks into single place

### PR DESCRIPTION
**Pull request by schnogz**
## Description
- check browser show user browser unsupported message immediately and in single place
- simplify app load failures via new `/#/app-error` route

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/scenes/AppError/AppError.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+const Error = ({ error }: Props) => (
+  <Wrapper>
+    {error === 'unsupportedBrowser' ? (
+      <>
+        <ErrorTitle>Unsupported Browser</ErrorTitle>
+        <ErrorSubtitle>
+          You are using {getBrowserNameAndVersion()}, a web browser that we do not support.
+        </ErrorSubtitle>
+        <ErrorSubtitle style={{ marginTop: '4px' }}>
+          Please update to the latest version of Chrome, Firefox, Safari or Opera.
```
Comment (pedroapfilho) [inline packages/blockchain-wallet-v4-frontend/src/scenes/AppError/AppError.tsx:45]: Could you add the versions?

File: packages/blockchain-wallet-v4-frontend/src/scenes/AppError/AppError.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        </ErrorSubtitle>
+      </>
+    ) : (
+      <>
+        <ErrorTitle>Application Error</ErrorTitle>
+        <ErrorSubtitle style={{ marginTop: '4px' }}>
+          We are having trouble retrieving essential data from our servers.
```
Comment (pedroapfilho) [inline packages/blockchain-wallet-v4-frontend/src/scenes/AppError/AppError.tsx:52]: "But rest assured your crypto is safe!"

**Review by milan-bc — APPROVED** [review]

File: packages/blockchain-wallet-v4-frontend/src/scenes/AppError/AppError.tsx (hunk 1 above)
Comment (schnogz) [reply packages/blockchain-wallet-v4-frontend/src/scenes/AppError/AppError.tsx:45]: yeah i do on line 42

File: packages/blockchain-wallet-v4-frontend/src/scenes/AppError/AppError.tsx (hunk 2 above)
Comment (schnogz) [reply packages/blockchain-wallet-v4-frontend/src/scenes/AppError/AppError.tsx:52]: good call

**Pull request merged**